libp2p/muxers/yamux/yamux.nim

```diff
@@ -273,17 +273,16 @@ method readOnce*(
         newLPStreamClosedError()
       else:
         newLPStreamConnDownError()
-  if channel.returnedEof:
+  if channel.isEof:
```

Comment:
As it's the only place where returnedEof was used, you can safely remove it from the `YamuxChannel` definition aswell.

Reply:
done